src/mscorlib/src/System/Threading/Tasks/Task.cs

```diff
@@ -1780,11 +1786,6 @@ internal void ScheduleAndStart(bool needsProtection)
                 // Queue to the indicated scheduler.
                 m_taskScheduler.InternalQueueTask(this);
             }
-            catch (ThreadAbortException tae)
-            {
-                AddException(tae);
-                FinishThreadAbortedTask(delegateRan: false);
-            }
```

Comment:
Let's leave the catch block in for now.  If we're going to remove it, we should make a pass through TPL and the ThreadPool to remove all of the associated support, but there were some concerns raised with doing so from Mono with regards to code sharing.  In my previous set of changes, I'd removed these, and then reverted.